packages/experimental/Overflow/src/OverflowItem/OverflowItem.tsx

```diff
@@ -0,0 +1,24 @@
+/** @jsxRuntime classic */
+/** @jsx withSlots */
+import * as React from 'react';
+import { View } from 'react-native';
+
+import { mergeProps, stagedComponent, withSlots } from '@fluentui-react-native/framework';
+
+import type { OverflowItemProps } from './OverflowItem.types';
+import { overflowItemName } from './OverflowItem.types';
+import { useOverflowItem } from './useOverflowItem';
+
+export const OverflowItem = stagedComponent<OverflowItemProps>((userProps: OverflowItemProps) => {
+  const { props, state } = useOverflowItem(userProps);
+  return (final: OverflowItemProps, ...children: React.ReactNode[]) => {
+    const merged = mergeProps(props, final);
+    if (!state.visible) {
+      return null;
+    }
+    return <View {...merged}>{children}</View>;
```

Comment:
><View [](http://example.com/codeflow?start=11&length=5)

What's the rationale behind wrapping the child in a View?

Reply:
Currently, OverflowItems style the width of themselves whenever a visible item needs to be truncated because of a lack of space. Could this be achieved via coning?

Reply:
It could, though you'd have to merge with the style prop of the child. I believe our mergeProps function takes care of that. You can look at the Shadow component as an example, search for cloneElement in the repo and you should be able to find examples.